$OpenBSD$

strlcat
strlcpy breaks the program if used with just sizeof(s)
- likely because sizeof returns size of the pointer
best working way so far is with size = MAX_INFO_STRING

Index: code/qcommon/q_shared.cpp
--- code/qcommon/q_shared.cpp.orig
+++ code/qcommon/q_shared.cpp
@@ -909,8 +909,8 @@ void Info_SetValueForKey( char *s, const char *key, co
 		return;
 	}
 
-	strcat (newi, s);
-	strcpy (s, newi);
+	strlcat (newi, s, sizeof(newi));
+	strlcpy (s, newi, MAX_INFO_STRING);
 }
 
 /*
@@ -1046,4 +1046,4 @@ void *Q_LinearSearch( const void *key, const void *ptr
 		ptr = (const char *)ptr + size;
 	}
 	return NULL;
-}
\ No newline at end of file
+}
